docs/documentation/concepts/index.md

```diff
@@ -6,29 +6,36 @@ layout: documentation
 
 # Concepts
 
-Eclipse SmartHome strictly differentiates between the physical view and the functional view on the system.
-While the physical view is required for setup, configuration, troubleshooting, etc., the functional side covers the information that matter to applications, such as user interfaces and automation logic.
+Eclipse SmartHome is built around two distinctly different views.
+The first is the physical view - devices, configuration, connections, etc.
+The second is the functional view.
+This functional view consists of how information about the physical view is presented in User Interfaces, how automation operates on representations of physical devices, or how an action on a User Interface affects the software associated with the physical device it represents.
 
 ## Things, Channels, Items and Links
 
-**Things** are the entities that can be physically added to a system and which can potentially provide many functionalities at once. It is important to note that things do not have to be devices, but they can also represent a web service or any other manageable source of information and functionality.
-Things provide their functionality through a set of **Channels**. Channels are "passive" and can be regarded as a declaration of a Thing, what it can offer. It is up to the individual setup, which of the Channels are actively used through Items (see below).
+**Things** are entities that can be physically added to a system.
+Things may provide more than one function (for example, a z-wave multi-sensor may provide a motion detector and also measure room temperature).
+It is important to note that Things do not have to be physical devices; they can also represent a web service or any other manageable source of information and functionality.
 
-**Items** represent (fine-grained) functionality that is used by applications - as user interfaces or automation logic. Items have a state and they can receive commands.
+Things provide their functionality through a set of **Channels**.  Channels are "passive" and can be regarded as a declaration of a Thing; what it can offer.
+Whether an installation takes advantage of a particular capability depends on whether it has been configured to do so.
+SmartHome makes use of Channels through **Links** you establish to Items when you configure your system.
```

Comment:
__Eclipse__ SmartHome [...]

Reply:
Agreed - this change will be in my next commit.

Reply:
Actually, I propose to drop ESH, since this does not make sense when the article is pulled into the OH documentation.  Leaving the reference in could confuse new users.